qualtran/bloqs/chemistry/chem_tutorials.py

```diff
@@ -0,0 +1,88 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Some utility functions for chemistry tutorials"""
+from typing import Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.optimize
+from numpy.typing import NDArray
+
+
+def linear(x, a, c):
+    return a * x + c
+
+
+def fit_linear(x, y):
+    try:
+        # pylint: disable-next=unbalanced-tuple-unpacking
```

Comment:
what is `disable-next`?

Reply:
disables for the next line only, which is maybe overkill here?

Reply:
ping @mpharrigan

Reply:
thanks for the info. what's the ping for?

Reply:
I was a little to eager to merge

Reply:
I had already approved it!